Restore all Travis tests

To ensure there are no schema or codelist errors before releasing 0.9.2.

I've restored the tests on [`155-restore-travis-tests`](https://github.com/open-contracting/infrastructure/tree/155-restore-travis-tests). They are currently failing on the `#ref` to the project schema in `project-package-schema.json` as follows:

```bash
=================================== FAILURES ===================================

/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/jsonref.py:229: jsonref.JsonRefError: JSONDecodeError: Expecting value: line 1 column 1 (char 0)

=============================== warnings summary ===============================

test_json.py::test_json_schema

  ERROR: schema/project-level/project-package-schema.json has JSONDecodeError: Expecting value: line 1 column 1 (char 0) at properties/projects/items
```
I can't see anything wrong with the package schema so I tried updating `test_json.py` to handle the project package schema in case that was causing the issue, but I'm still getting errors. My efforts are on [this branch](https://github.com/open-contracting/standard-maintenance-scripts/tree/project-package).

@jpmckinney could you take a look please?

--- a/.travis.yml
+++ b/.travis.yml
@@ -23,5 +23,5 @@ install:
   - curl -s -S --retry 3 $BASEDIR/tests/install.sh | bash -
   - pip install -r requirements.txt
 script:
-#  - curl -s -S --retry 3 $BASEDIR/tests/script.sh | bash -
+  - curl -s -S --retry 3 $BASEDIR/tests/script.sh | bash -
   - make

--- a/schema/project-level/project-schema.json
+++ b/schema/project-level/project-schema.json
@@ -82,7 +82,8 @@
       "enum": [
         "construction",
         "rehabilitation",
-        "replacement"
+        "replacement",
+        "expansion"
       ],
       "codelist": "projectType.csv",
       "openCodelist": false

